Makefile

```diff
@@ -149,6 +149,9 @@ indent:
 .PHONY : pre_commit_check
 pre_commit_check: all indent clean
 
+dev:
+	@docker run -it --rm -v `pwd`:/home/s2n-dev/s2n camshaft/s2n-dev:1.0.0
```

Comment:
I've been adding ` --ulimit memlock=-1` to get the prlimit stuff to work.

Reply:
Ah that's a good call